website/source/docs/agent/options.html.md

```diff
@@ -261,9 +344,13 @@ will exit with an error at startup.
 
   The only permission needed is the ListAll method for NetworkInterfaces. It is recommended you make a dedicated key used only for auto-joining.
 
+  As of Consul 0.9.1 this parameter has been deprecated. See [-retry-join](#_retry_join) for details.
+
 * <a name="_retry_join_azure_tag_value"></a><a href="#_retry_join_azure_tag_value">`-retry-join-azure-tag-value`
   </a> - The Azure instance tag value to filter on.
 
+  As of Consul 0.9.1 this parameter has been deprecated. See [-retry-join](#_retry_join) for details.
```

Comment:
There should be a comma after `As of Consul 0.9.1`, or you could rewrite the sentence as "This parameter has been deprecated as of Consul 0.9.1"